RHEL/6/input/fixes/bash/disable_ctrlaltdel_reboot.sh

```diff
@@ -0,0 +1,5 @@
+if [ -f /etc/init/control-alt-delete.conf ]; then
```

Comment:
@shawndwells shouldn't it be `if [ ! -f /etc/init/control-alt-delete.override ]`?